### ISSUE TITLE
More Confirmed iGPUs/dGPUS

### ISSUE THREAD

**Issue by 5T33Z0** [notjosh/AdvancedMap#10]
AdvancedMap working fine in Sonoma on the following iGPUs/dGPUS

Lenovo T530, with Intel HD 4000 (patched with OCLP):
 
![T530](https://github.com/user-attachments/assets/3f6d0d1f-442b-4bc0-b8c0-ccf1af5a8826)

Lenovo T490, with Intel UHD 620:

![T490](https://github.com/user-attachments/assets/f3018c71-8d82-4288-bece-adfcfabc1cbb)

iMac12,2 with NVIDIA QUADRO K1100m (patched with OCLP):

![imac12,2](https://github.com/user-attachments/assets/5af02f2e-97b0-4503-9c05-d5fd6093219a)

**Comment by bootlegseahorse** [comment]
Does the 2011 even support Metal? So does it mean that AdvancedMap doesn't even need Metal support?

**Comment by 5T33Z0** [comment]
> Does the 2011 even support Metal? So does it mean that AdvancedMap doesn't even need Metal support?

No, because it uses an AMD Terrascale card. But I upgraded mine with an NVIDIA Quadro K1100m.

**Comment by 5T33Z0** [comment]
GA-Z490 Vision with Intel I9-10850K and AMD RX580 4GB using AdvanceMap 4.0:

![Bildschirmfoto 2024-08-09 um 17 17 21](https://github.com/user-attachments/assets/764ba522-6e4b-4280-aad9-bf843c5c9a1e)

**Comment by bootlegseahorse** [comment]
Is detailed map working perfectly on all these GPUs too? 

It used to crash on all my Intel Macs (#6) but is working perfectly now without crashes.

I wonder if it's the same case with your Macs.

**Comment by 5T33Z0** [comment]
<img width="1365" height="793" alt="Image" src="https://github.com/user-attachments/assets/d9348f0a-2fc5-4a86-b724-37a826e4ed1a" />

And another one confirmed Working. This is kinda special, because this is a Ryzentosh Laptop with Radeon RX Vega graphics running macOS Tahoe. NootedRed.kext drives the iGPU
